stage-prep/06-automount/files/media-automount

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+#$1 = <dev>
+
+# Default options to use for mounting
+AUTOMOUNT_OPTS='users'
+# Default type to use for mounting
+AUTOMOUNT_TYPE='auto'
+
+# Directory to look for type-specific settings
+confdir=/etc/udev/media-automount.d
+
+# Directory to use as parent media dir for mountpoints
+mediadir=/media
+
+[ $(id -u) != 0 ] && {
+    echo "This tool requires root permissions"
```

Comment:
Is there a way to make this user available? Some different permissions? Or is it just because `mount` requires sudo.  Can the sudoers file be changed to accommodate this

Reply:
yeah, `mount` needs sudo